src/lib/containers/markdown/MarkdownSynapse.tsx

```diff
@@ -761,16 +576,23 @@ export default class MarkdownSynapse extends React.Component<
     const { renderInline } = this.props
     const { isLoading, error } = this.state
 
+    const wikiContext: SynapseWikiContextType = {
+      ownerId: this.props.ownerId,
+      wikiId: this.props.wikiId,
+      wikiPage: this.state.data,
+      fileHandles: this.state.fileHandles,
+    }
```

Comment:
This data seemed to me like it should be passed through context instead of props, since it's all wiki-related, and markdown may not necessarily come from a wiki.

I don't think it's perfect but trying to keep the refactor small here